src/components/box/box.stories.mdx

```diff
@@ -63,11 +63,37 @@ Basic component that gives access to a number of styling options. Should be used
   </Story>
 </Preview>
 
-## Props
+## Scroll
+Use the `scrollType` prop to indicate which theme scrollbar you would like to display. Use `light` or `dark` values to display as below. This will still require the use of the `overflow` prop and only displays in Chrome or Safari
```

Comment:
We use a `variant` in most of new components to change the theme, should this be `scrollVariant`?